src/Microsoft.AspNetCore.SignalR.Client.TS/TransportFactory.ts

```diff
@@ -0,0 +1,23 @@
+import { ITransport, WebSocketTransport, ServerSentEventsTransport, LongPollingTransport } from "./Transports"
```

Comment:
I don't like this being exposed as a separate module. I don't think it is of any use outside the connection (that's why createTransport was private)